test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -394,6 +417,42 @@ instance
             , (5, pure $ ApiMnemonicT y)
             ]
 
+instance Arbitrary ApiBlockData where
+    arbitrary = genericArbitrary
+    shrink = genericShrink
+
+instance Arbitrary SlotId where
+    arbitrary = SlotId <$> arbitrary <*> arbitrary
+    shrink = genericShrink
+
+instance Arbitrary ApiCoinSelection where
+    arbitrary = genericArbitrary
+    shrink = genericShrink
+
+instance Arbitrary PostTransactionData where
+    arbitrary = genericArbitrary
+    shrink = genericShrink
+
+instance Arbitrary ApiTransaction where
+    arbitrary = genericArbitrary
+    shrink = genericShrink
+
+instance Arbitrary (Quantity "block" Natural) where
+    shrink (Quantity 0) = []
+    shrink _ = [Quantity 0]
+    arbitrary = Quantity . fromIntegral <$> (arbitrary @Word8)
+
+instance Arbitrary (Hash "Tx") where
+    arbitrary = Hash . B8.pack <$> replicateM 64 arbitrary
```

Comment:
huge entropy here - do we want to give less entropy then 64 bytes for constructing `Hash "Tx"` in order to trigger collisions?